Backend/DynamicQR/DynamicQR.Api/EndPoints/QrCodeTargets/QrCodeTargetPut/QrCodeTargetPut.cs

```diff
@@ -32,7 +33,16 @@ public async Task<HttpResponseData> RunAsync([HttpTrigger(AuthorizationLevel.Fun
 
         Application.QrCodes.Commands.UpdateQrCodeTarget.Command? coreCommand = request.Result.ToCore(id);
 
-        Application.QrCodes.Commands.UpdateQrCodeTarget.Response coreResponse = await _mediator.Send(coreCommand);
+        Application.QrCodes.Commands.UpdateQrCodeTarget.Response coreResponse;
+
+        try
+        {
+            coreResponse = await _mediator.Send(coreCommand);
+        }
+        catch (StorageException)
+        {
+            return await CreateJsonResponse(req, null, HttpStatusCode.BadGateway);
```

Comment:
- er wordt een JsonResponse teruggegeven, maar je wilt een simpele Response met Status code teruggeven
- er mist een response message, het is goed als user om te weten wat er ongeveer mis is gegaan, dus in dit geval iets van 'er ging iets mis tijdens het benaderen van de database + ex.Message'
- Je hebt een response toegevoegd, maar dit niet in de API definitie gezet, dus er moet nog een [OpenApi] attribuut toegevoegd worden voor de BadGateway response. Je kan ook een OpenApiResponseBadGateway attribuut maken

En dit dan voor alle functions waar deze code is toegevoegd.